vector/src/main/java/im/vector/app/features/call/VectorCallActivity.kt

```diff
@@ -114,7 +121,9 @@ class VectorCallActivity : VectorBaseActivity<ActivityCallBinding>(), CallContro
         if (intent.getStringExtra(EXTRA_MODE) == INCOMING_RINGING) {
             turnScreenOnAndKeyguardOff()
         }
-
+        if (savedInstanceState != null) {
```

Comment:
do that in the fragment onAttached?

Reply:
What does it change, I still need to access the callback